### PR TITLE
Quiz patch + Sample Exam 14Q complete

### PR DESCRIPTION
**Pull request by ctrlcade**
All 9 Quizzes & their solutions complete. I've also done 14 of the sample exam questions with their solutions, the only question not done is the query optimisation one. Please merge this into the main branch so everyone else can see!

### CODE REVIEW

**Comment by Rqcker** [comment]
Hi Cade @ctrlcade,
  This is great, I read the answer you wrote, it is very credible. I hope you can complete the query optimization part tomorrow. Thank you for your contribution to the sophomore year. We will merge your answers in time.

Junhao

File: quizzes/Quiz-9-Concurrency-Control.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- A. Record controller
+- B. Two phase lock
+- C. Exclusive lock
+- D. Authorization rule
+
+Answer - B. Two phase lock
+
+---
+
+Q9 - A deadlock exists in the system if and only if the wait-for graph contains a(an) ____
```
Comment (Rqcker) [inline quizzes/Quiz-9-Concurrency-Control.md:89]: Answer of Q9 need check

**Review by Rqcker — COMMENTED** [review]
Q9 need check

File: quizzes/Quiz-9-Concurrency-Control.md (hunk 1 above)
Comment (ctrlcade) [reply quizzes/Quiz-9-Concurrency-Control.md:89]: what seems to be the problem? checked it and it is coreect

**Review by TheIbrahimMalik — APPROVED** [review]
Good Effort!

**Review by TheIbrahimMalik — COMMENTED** [review]
Good stuff

File: quizzes/Quiz-9-Concurrency-Control.md (hunk 1 above)
Comment (Rqcker) [reply quizzes/Quiz-9-Concurrency-Control.md:89]: Yeah, my bad. That should be Cycle.

**Review by Rqcker — COMMENTED** [review]
Fixed the Cycle Q&A

**Pull request merged**